src/library/borrowbook/BorrowBookUI.java

```diff
@@ -4,115 +4,115 @@
 
 public class BorrowBookUI {
 	
-	public static enum uI_STaTe { INITIALISED, READY, RESTRICTED, SCANNING, IDENTIFIED, FINALISING, COMPLETED, CANCELLED };
+    public static enum UiState {INITIALISED, READY, RESTRICTED, SCANNING, IDENTIFIED, FINALISING, COMPLETED, CANCELLED};
 
-	private bORROW_bOOK_cONTROL CoNtRoL;
-	private Scanner InPuT;
-	private uI_STaTe StaTe;
+    private BorrowBookControl control;
+    private Scanner input;
+    private UiState state;
 
 	
-	public BorrowBookUI(bORROW_bOOK_cONTROL control) {
-		this.CoNtRoL = control;
-		InPuT = new Scanner(System.in);
-		StaTe = uI_STaTe.INITIALISED;
-		control.SeT_Ui(this);
-	}
+    public BorrowBookUI(BorrowBookControl control) {
+        this.control = control;
+        input = new Scanner(System.in);
+        state = UiState.INITIALISED;
+        control.setUi(this);
+    }
 
 	
-	private String iNpUT(String PrOmPt) {
-		System.out.print(PrOmPt);
-		return InPuT.nextLine();
-	}	
+    private String input(String prompt) {
+        System.out.print(prompt);
+        return input.nextLine();
+    }	
 		
 		
-	private void OuTpUt(Object ObJeCt) {
-		System.out.println(ObJeCt);
-	}
+    private void output(Object object) {
+        System.out.println(object);
+    }
 	
 			
-	public void SeT_StAtE(uI_STaTe StAtE) {
-		this.StaTe = StAtE;
-	}
+    public void setState(UiState state) {
+        this.state = state;
+    }
 
 	
-	public void RuN() {
-		OuTpUt("Borrow Book Use Case UI\n");
+    public void run() {
+        output("Borrow Book Use Case UI\n");
 		
-		while (true) {
+        while (true) {
 			
-			switch (StaTe) {			
+            switch (state) {			
 			
-			case CANCELLED:
-				OuTpUt("Borrowing Cancelled");
-				return;
+            case CANCELLED:
+                output("Borrowing Cancelled");
+                return;
 
 				
-			case READY:
-				String MEM_STR = iNpUT("Swipe member card (press <enter> to cancel): ");
-				if (MEM_STR.length() == 0) {
-					CoNtRoL.CaNcEl();
-					break;
-				}
-				try {
-					int MeMbEr_Id = Integer.valueOf(MEM_STR).intValue();
-					CoNtRoL.SwIpEd(MeMbEr_Id);
-				}
-				catch (NumberFormatException e) {
-					OuTpUt("Invalid Member Id");
-				}
-				break;
+            case READY:
+                String memberString = input("Swipe member card (press <enter> to cancel): ");
+                if (memberString.length() == 0) {
+                    control.cancelOperation();
+                    break;
+                }
+                try {
+                    int memberId = Integer.valueOf(memberString).intValue();
+                    control.cardSwiped(memberId);
+                }
+                catch (NumberFormatException exit) {

```

Comment:
consider changing exit to exception

Reply:
Resolution: https://github.com/9Mcgusto6/ITC205-Team17/pull/6/commits/38d55bba36e203d850141f49122bcc9d8d7af203